File: BuildVariants/app/src/free/java/github/nisrulz/sample/buildvariants/Logger.java
package github.nisrulz.sample.buildvariants;

import android.util.Log;

public class Logger {
  public static final String LOGTAG = "FreeBuild---------->";

  public static void logd(String message) {
    Log.d(LOGTAG, message);
  }

  public static void loge(Exception ex) {
    Log.d(LOGTAG, "Exception", ex);
  }
}

File: BuildVariants/app/src/free/java/github/nisrulz/example/buildvariants/Logger.kt
package github.nisrulz.example.buildvariants

import android.util.Log

object Logger {
    private const val LOGTAG = ">> FreeBuild---------->"

    fun logd(message: String = "") {
        Log.d(LOGTAG, message)
    }

    fun loge(ex: Exception?) {
        Log.d(LOGTAG, "Exception", ex)
    }
}